resources/css/bem/beatmapset-discussions-chart.less

```diff
@@ -71,31 +71,31 @@
     }
 
     &--mapper-note {
-      --icon-colour: @beatmap-discussion--color-mapper-note;
+      --icon-colour: var(--beatmapset-discussion-colour--mapper-note);
```

Comment:
wrong case 👀

Reply:
somehow I corrected the names on the branch I was rebasing but not this one 🤨